docs/versionIndex.md

```diff
@@ -44,6 +45,17 @@ backend services have been tested with each released version of ADF.
 -   [v2.1.0](#v210)
 -   [v2.0.0](#v200)
 
+## v6.6.0
```

Comment:
I think it should be `6.7.0` as this will be the version that will be released soon